### PR TITLE
[docker] fixes #2072 - LABELS in final Docker image (not temporary)

### LINKED ISSUE #2072: Fix labels of mainnet Docker image
In [mainnet Dockerfile](docker/images/mainnet/Dockerfile) labels are set in the temporary image that's discarded rather than final image .

### PR DESCRIPTION
**Pull request by olemis**
Fixes #2072 . Supersedes #2074 

Changes:
- LABELS in final Docker image (not temporary)

Does this change need to mentioned in CHANGELOG.md?
no

### CODE REVIEW

**Comment by olemis** [comment]
@stdevStark you are still required to sign the CLA

**Comment by stdevStark** [comment]
@olemis can you revise if [labels](https://microbadger.com/images/simelotech/skycoin:feature-stdevStark-2072) are correct ?

File: docker/images/mainnet/Dockerfile [hunk 1]
```diff
@@ -54,6 +40,21 @@ COPY docker_launcher.sh /tmp/files/usr/local/bin/docker_launcher.sh
...
+# Image labels
+LABEL "org.label-schema.name"="Skycoin" \
+      "org.label-schema.description"="Skycoin core docker image" \
+      "org.label-schema.vcs-url"="https://github.com/skycoin/skycoin/tree/develop/docker/images/mainnet" \
+      "org.label-schema.vendor"="Skycoin project" \
+      "org.label-schema.url"="skycoin.net" \
+      "org.label-schema.schema-version"="1.0" \
+      "org.label-schema.build-date"=$BDATE \
+      "org.label-schema.vcs-ref"=$SCOMMIT \
+      "org.label-schema.version"="1.0.0-rc.1" \
```
Comment (olemis) [inline docker/images/mainnet/Dockerfile]: Should be Skycoin version if building from a tag .

Comment (stdevStark) [reply docker/images/mainnet/Dockerfile]: This is made in #2071

**Comment by olemis** [comment]
I'll close this PR since #2076 will fix #2072 as well .

**Pull request closed without merging**